Documentation/README.md

```diff
@@ -1,12 +1,12 @@
 # Resources
 See the following resources to get started.
 
-## .NET Core
-* [.NET Core Documentation](https://docs.microsoft.com/dotnet/)
-* [Install .NET Core on Raspberry Pi](https://github.com/dotnet/core/blob/master/samples/RaspberryPiInstructions.md)
-* [.NET Core ARM64 Status](https://github.com/dotnet/announcements/issues/82)
-* [.NET Core Docker Samples](https://github.com/dotnet/dotnet-docker/tree/master/samples)
-* [How to Prepare a Publish Profile](How-to-Deploy-an-IoT-App.md)
+## Repo Layout
+
+This repository mainly contains two different components:
+
+1. **System.Device.Gpio** library and tests which is the main library that has the implementation for protocols such as: GPIO, SPI, I2C, PWM. This library is fully supported by the dotnet team since it has the same level of support that dotnet/corefx does. All the code for the library lives under src/System.Device.Gpio. This library targets .NET Standard 2.0, and will work on both Windows and Linux. It's implementation consists of just IL code, so that means that it is supported across different platforms. In order to add new API to this library, an API Proposal would have to be submitted and approved first. [Here](https://github.com/dotnet/iot/issues/122) is an example of a how a good API proposal should look like.
+1. **IoT.Device.Bindings** device bindings library. These is a collection of types which work as wrappers (or bindings) for devices and sensors which are able to talk to a microcontroller unit (or MCU like a Raspberry Pi for example) using the protocols supported by System.Device.Gpio. For example: [BME280](/../src/devices/Bmxx80/README.md) is a temperature sensor which uses SPI and I2C in order to communicate with a MCU and is able to report the current temperature. Because the process of how to compute the temperature from the data is not trivial, we have a `Bme280` class which exposes friendly methods like `ReadTemperature()` which will internally use either SPI or I2C to get the current temperature value. In order to start adding a new binding, check out our [guide on how to contribute a new binding](/../tools/templates/DeviceBindingTemplate/README.md).
```

Comment:
might be worth to mention that src/Iot.Device.Bindings contains the project but src/devices is the actual implementation